addons/sale_timesheet/models/sale_order.py

```diff
@@ -15,7 +15,7 @@ class SaleOrder(models.Model):
     timesheet_count = fields.Float(string='Timesheet activities', compute='_compute_timesheet_count', groups="hr_timesheet.group_hr_timesheet_user")
 
     # override domain
-    project_id = fields.Many2one(domain="[('pricing_type', '!=', 'employee_rate'), ('analytic_account_id', '!=', False), ('company_id', '=', company_id)]")
+    project_id = fields.Many2one(domain="[('pricing_type', '!=', 'employee_rate'), ('analytic_account_id', '!=', False), '|', ('company_id', '=', company_id), ('company_id', '=', False)]")
```

Comment:
```suggestion
    project_id = fields.Many2one(domain="[('pricing_type', '!=', 'employee_rate'), ('analytic_account_id', '!=', False)]", check_company=True)
```
why not using check_company? 🤔

Reply:
ohhh I forgot that attribute, I updated it, thanks 😸